packages/react-core/src/components/DualListSelector/DualListSelector.tsx

```diff
@@ -86,14 +104,19 @@ export class DualListSelector extends React.Component<DualListSelectorProps, Dua
     removeSelectedAriaLabel: 'Remove selected',
     removeAllAriaLabel: 'Remove all'
   };
+  private originalCopy = this.props.availableOptions;
```

Comment:
I think I do need it because availableOptions can be re-passed in after a list update (like add/remove). To maintain the overall order I need an initial array.

Reply:
Would it be better to make a separate prop to use instead? Like `initialOptions`?

Reply:
oh, I am thinking that originalCopy is just a pointer to prop.availableOptions, so any changes to one should reflect in the other. Just would be cleaner not to have 3 things in code tracking availableOptions is all. But it's only a nit so if it causes issues it's okay the way it is
```
originalCopy
props.availableOptions
state.availableOptions
```